server/src/test/java/io/crate/planner/node/ddl/RestoreSnapshotPlanTest.java

```diff
@@ -1,137 +0,0 @@
-/*
- * Licensed to Crate.io GmbH ("Crate") under one or more contributor
- * license agreements.  See the NOTICE file distributed with this work for
- * additional information regarding copyright ownership.  Crate licenses
- * this file to you under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.  You may
- * obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * However, if you have executed another commercial license agreement
- * with Crate these terms will supersede the license and you may use the
- * software solely pursuant to the terms of the relevant commercial agreement.
- */
-
-package io.crate.planner.node.ddl;
-
-import static io.crate.metadata.PartitionName.templateName;
-import static io.crate.testing.Asserts.assertThat;
-
-import java.util.List;
-import java.util.Set;
-import java.util.UUID;
-
-import org.elasticsearch.snapshots.SnapshotId;
-import org.elasticsearch.snapshots.SnapshotInfo;
-import org.junit.Test;
-
-import io.crate.analyze.BoundRestoreSnapshot;
-import io.crate.metadata.RelationName;
-import io.crate.metadata.Schemas;
-
-public class RestoreSnapshotPlanTest {
-
-    @Test
-    public void testResolveTableIndexWithIgnoreUnavailable() throws Exception {
-        var context = RestoreSnapshotPlan.resolveIndexNames(
-            "my_repo",
-            Set.of(new BoundRestoreSnapshot.RestoreTableInfo(
-                new RelationName(Schemas.DOC_SCHEMA_NAME, "my_table"), null)),
-            true,
-            null
-        ).get();
-
-        assertThat(context.resolvedIndices()).containsExactlyInAnyOrder(
-                "my_table",
-                templateName(Schemas.DOC_SCHEMA_NAME, "my_table") + "*");
-        assertThat(context.resolvedTemplates()).containsExactly(".partitioned.my_table.");
-    }
-
-    @Test
-    public void testResolveTableIndexFromSnapshot() {
-        var context = new RestoreSnapshotPlan.ResolveIndicesAndTemplatesContext();
-        RestoreSnapshotPlan.resolveTableFromSnapshots(
-            new BoundRestoreSnapshot.RestoreTableInfo(new RelationName("custom", "restoreme"), null),
-            List.of(
-                new SnapshotInfo(
-                    new SnapshotId("snapshot01", UUID.randomUUID().toString()),
-                    List.of("custom.restoreme"),
-                    0L,
-                    false)),
-            context
-        );
-
-        assertThat(context.resolvedIndices()).containsExactly("custom.restoreme");
-        assertThat(context.resolvedTemplates()).isEmpty();
-    }
```

Comment:
We already have similar test case covering new code (added in https://github.com/crate/crate/pull/15095).